blog.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Adoptable Dogs Page </title>
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+</head>
+<body>
+    <script src="script/script.js"> </script>
+  <header>
+    <div class="header">
+      <img class= "dogface" src="C:/Users/JoJo/Desktop/html200/adopt-a-dog-spring2019/images/header-image.jpg" alt= "brown dog looking through cracked blinds with sunlight beaming through">
+           <img class="logo" src="C:/Users/JoJo/Desktop/html200/adopt-a-dog-spring2019/images/adoptadog-logo.svg" alt= "adoptadog white logo">
+        <h2 class="logo" style="color:rgb(132,187,66);"> Seattle's premiere dog adoption agency </h2>
+    </div>
+    </header>
+
+    <div class="buttons">
+      <nav>
+           <a href="index.html">Home  /</a>
+        </nav>
+          <nav>
+              <a href="dogs.html">Dogs  /</a>
+              </nav>
+          <nav>
+              <a href="blog.html">Blog</a>
+              </nav>
+      </div>
+      <section class= "blog">
+        <div>
+           <h2 style="color:rgb(9,60,96);">Adoptadog Blog</h2>
+         <br>
+       </div>
+         <section class= "Travel">
+ 	          <h2 style="color:rgb(9,60,96);" >Traveling with your Dog</h2>
+           <img class= "blogpic" src="C:/Users/JoJo/Desktop/html200/adopt-a-dog-spring2019/images/blog-1.jpg" alt= "dirty blond girl sitting on canyon cliff with dog">
+          <p>
+           Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Morbi tincidunt ornare massa eget egestas. Purus non enim praesent elementum facilisis leo vel. Ultricies mi eget mauris pharetra et ultrices neque ornare aenean. Mauris sit amet massa vitae tortor condimentum. Tellus in hac habitasse platea dictumst. Tortor aliquam nulla facilisi cras fermentum. In dictum non consectetur a erat nam at. Nunc faucibus a pellentesque sit amet porttitor eget. Ipsum dolor sit amet consectetur adipiscing elit ut aliquam purus.
+           </p>
+          <p> Duis ultricies lacus sed turpis tincidunt id. Ipsum a arcu cursus vitae congue. Ipsum consequat nisl vel pretium lectus. A diam maecenas sed enim ut sem viverra. Nulla facilisi nullam vehicula ipsum a arcu cursus vitae. Dictum non consectetur a erat nam at lectus urna. Et molestie ac feugiat sed lectus vestibulum mattis. Scelerisque fermentum dui faucibus in. At imperdiet dui accumsan sit amet nulla facilisi morbi. Dictumst quisque sagittis purus sit amet volutpat consequat. Sagittis orci a scelerisque purus semper. Scelerisque varius morbi enim nunc faucibus a. Commodo elit at imperdiet dui accumsan sit amet nulla. Morbi quis commodo odio aenean sed adipiscing diam.
+         </p>
+          </section>
+          <section class= "Walk-Multiple">
```

Comment:
It's generally good practice to use all lowercase letters for CSS classes with a - seperating the words. Also, watch for extra spaces. You don't need  a space after the = sign.

```
<section class="walk-multiple">
```